### ISSUE TITLE
No interaction with panel possible when start menu is open

### ISSUE THREAD

**Issue by Joshiy13** [linuxmint/cinnamon#12284]
### Distribution

Linux Mint 21.3 Cinnamon

### Package version

6.0.4

### Frequency

Always

### Bug description

When you press your key for your start menu, you can't use your panel. For example, you are playing a game in fullscreen, press the menu button and try to open an application from your panel. It just closes the start menu and doesnt open the application that is clicked on. 
![image](https://github.com/linuxmint/cinnamon/assets/79088559/b9b6f10c-e3b8-4dda-94ab-0b5d2c0328f8)

### Steps to reproduce

Press the Key for the Start Menu, then try to click on any app Icons in the panel

### Expected behavior

When I click on the Panel with the Start menu open it should start that application. Not close the menu and then dont do anything.

**Comment by fredcw** [comment]
Duplicate of https://github.com/linuxmint/cinnamon/issues/11658

**Comment by Joshiy13** [comment]
> Duplicate of #11658

Not entirely, as this also happens when no app is running in fullscreen. I will update the description accordingly

**Comment by fredcw** [comment]
Yes, MATE desktop works like this. I can't say for sure but I suspect that this would take a significant amount of work since Cinnamon wasn't designed to work that way.

**Comment by JosephMcc** [comment]
This isn't a bug. It's just part of how cinnamon works. When you open a menu like that, for all intensive purposes, it grabs the focus. It doesn't give up that focus until you close the menu. You will notice the same behavior if you open a terminal window, open a menu on the menubar, and then click an icon on the Cinnamon panel. The menu from the terminal has grabbed the focus until you close it. That second click outside the menu is taken to close it.

**Comment by fredcw** [comment]
@JosephMcc 

>intensive purposes

Lol, are you using voice recognition?

**Comment by JosephMcc** [comment]
@fredcw Lol. No. Just not awake enough and being dumb.
